### PR TITLE
Add DaNe dataset

### PR DESCRIPTION
**Pull request by ophelielacroix**

### CODE REVIEW

**Comment by yjernite** [comment]
Thanks, this looks great!

For the code quality test, it looks like `flake8` is throwing the error, so you can tun `flake8 datasets` locally and fix the errors it points out until it passes

File: datasets/dane/dane.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+_HOMEPAGE = "https://github.com/alexandrainst/danlp/blob/master/docs/docs/datasets.md#dane"
+
+_LICENSE = "CC BY-SA 4.0"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://danlp.alexandra.dk/304bd159d5de/datasets/ddt.zip"
+
+
+class DaNE(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/dane/dane.py:66]:
```suggestion
class Dane(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `dane`

File: datasets/dane/dane.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    """DaNE dataset"""
+
+    def _info(self):
+        features = datasets.Features(
```
Comment (lhoestq) [inline datasets/dane/dane.py:70]: nice features !

File: datasets/dane/dataset_infos.json [hunk 3]
```diff
@@ -0,0 +1 @@
..., "ccomp", "amod", "cc", "advcl", "nsubj", "punct", "case"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}, "ner_tags": {"feature": {"num_classes": 9, "names": ["O", "B-PER", "I-PER", "B-ORG", "I-ORG", "B-LOC", "I-LOC", "B-MISC", "I-MISC"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "da_ne", "config_name": "default", "version": {"version_str": "0.0.0", "description": null, "major": 0, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 7311252, "num_examples": 4384, "dataset_name": "da_ne"}, "test": {"name": "test", "num_bytes": 909739, "num_examples": 566, "dataset_name": "da_ne"}, "validation": {"name": "validation", "num_bytes": 940453, "num_examples": 565, "dataset_name": "da_ne"}}, "download_checksums": {"https://danlp.alexandra.dk/304bd159d5de/datasets/ddt.zip": {"num_bytes": 1209710, "checksum": "df97f3eaa396fd52bf35060c63960aebefaa47e5e6125fb75fe3be098384ebd2"}}, "download_size": 1209710, "post_processing_size": null, "dataset_size": 9161444, "size_in_bytes": 10371154}}
```
Comment (lhoestq) [inline datasets/dane/dataset_infos.json:1]:
```suggestion
{"default": {"description": "The DaNE dataset has been annotated with Named Entities for PER, ORG and LOC \nby the Alexandra Institute.\nIt is a reannotation of the UD-DDT (Universal Dependency - Danish Dependency Treebank) \nwhich has annotations for dependency parsing and part-of-speech (POS) tagging. \nThe Danish UD treebank (Johannsen et al., 2015, UD-DDT) is a conversion of \nthe Danish Dependency Treebank (Buch-Kromann et al. 2003) based on texts \nfrom Parole (Britt, 1998).\n", "citation": "@inproceedings{hvingelby-etal-2020-dane,\n    title = \"{D}a{NE}: A Named Entity Resource for {D}anish\",\n    author = \"Hvingelby, Rasmus  and\n      Pauli, Amalie Brogaard  and\n      Barrett, Maria  and\n      Rosted, Christina  and\n      Lidegaard, Lasse Malm  and\n      S{\\o}gaard, Anders\",\n    booktitle = \"Proceedings of the 12th Language Resources and Evaluation Conference\",\n    month = may,\n    year = \"2020\",\n    address = \"Marseille, France\",\n    publisher = \"European Language Resources Association\",\n    url = \"https://www.aclweb.org/anthology/2020.lrec-1.565\",\n    pages = \"4597--4604\",\n    abstract = \"We present a named entity annotation for the Danish Universal Dependencies treebank using the CoNLL-2003 annotation scheme: DaNE. It is the largest publicly available, Danish named entity gold annotation. We evaluate the quality of our annotations intrinsically by double annotating the entire treebank and extrinsically by comparing our annotations to a recently released named entity annotation of the validation and test sections of the Danish Universal Dependencies treebank. We benchmark the new resource by training and evaluating competitive architectures for supervised named entity recognition (NER), including FLAIR, monolingual (Danish) BERT and multilingual BERT. We explore cross-lingual transfer in multilingual BERT from five related languages in zero-shot and direct transfer setups, and we show that even with our modestly-sized training set, we improve Danish NER over a recent cross-lingual approach, as well as over zero-shot transfer from five related languages. Using multilingual BERT, we achieve higher performance by fine-tuning on both DaNE and a larger Bokm{\u0007a}l (Norwegian) training set compared to only using DaNE. However, the highest performance isachieved by using a Danish BERT fine-tuned on DaNE. Our dataset enables improvements and applicability for Danish NER beyond cross-lingual methods. We employ a thorough error analysis of the predictions of the best models for seen and unseen entities, as well as their robustness on un-capitalized text. The annotated dataset and all the trained models are made publicly available.\",\n    language = \"English\",\n    ISBN = \"979-10-95546-34-4\",\n}\n", "homepage": "https://github.com/alexandrainst/danlp/blob/master/docs/docs/datasets.md#dane", "license": "CC BY-SA 4.0", "features": {"sent_id": {"dtype": "string", "id": null, "_type": "Value"}, "text": {"dtype": "string", "id": null, "_type": "Value"}, "tok_ids": {"feature": {"dtype": "int64", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "tokens": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "lemmas": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "pos_tags": {"feature": {"num_classes": 17, "names": ["NUM", "CCONJ", "PRON", "VERB", "INTJ", "AUX", "ADJ", "PROPN", "PART", "ADV", "PUNCT", "ADP", "NOUN", "X", "DET", "SYM", "SCONJ"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}, "morph_tags": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "dep_ids": {"feature": {"dtype": "int64", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "dep_labels": {"feature": {"num_classes": 36, "names": ["parataxis", "mark", "nummod", "discourse", "compound:prt", "reparandum", "vocative", "list", "obj", "dep", "det", "obl:loc", "flat", "iobj", "cop", "expl", "obl", "conj", "nmod", "root", "acl:relcl", "goeswith", "appos", "fixed", "obl:tmod", "xcomp", "advmod", "nmod:poss", "aux", "ccomp", "amod", "cc", "advcl", "nsubj", "punct", "case"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}, "ner_tags": {"feature": {"num_classes": 9, "names": ["O", "B-PER", "I-PER", "B-ORG", "I-ORG", "B-LOC", "I-LOC", "B-MISC", "I-MISC"], "names_file": null, "id": null, "_type": "ClassLabel"}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "dane", "config_name": "default", "version": {"version_str": "0.0.0", "description": null, "major": 0, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 7311252, "num_examples": 4384, "dataset_name": "dane"}, "test": {"name": "test", "num_bytes": 909739, "num_examples": 566, "dataset_name": "dane"}, "validation": {"name": "validation", "num_bytes": 940453, "num_examples": 565, "dataset_name": "dane"}}, "download_checksums": {"https://danlp.alexandra.dk/304bd159d5de/datasets/ddt.zip": {"num_bytes": 1209710, "checksum": "df97f3eaa396fd52bf35060c63960aebefaa47e5e6125fb75fe3be098384ebd2"}}, "download_size": 1209710, "post_processing_size": null, "dataset_size": 9161444, "size_in_bytes": 10371154}}
```

`da_ne` -> `dane`

**Review by lhoestq — COMMENTED** [review]
Looking pretty good so far !
Thanks for adding this one 

Don't forget to add the tags of the dataset card. It's the yaml part at the top of the dataset card.
More info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

**Pull request closed without merging**